### DISCUSSION TITLE
GitHub Mobile Actions/Workflows status.

### DISCUSSION THREAD

**Discussion by spaluchiewicz** [community/community#32872] [Mobile] [answered]
Hi GitHub Mobile Team.
Thanks for putting all your effort to make this app. I think it could support users more with additional feature for Actions/Workflows status view. Look forward to hear your opinion about that.

Best Regards, Sebastian

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.79.0 (77425110)
Device: iPhone 13 Pro (iOS 15.6.1)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Thanks for the feedback! I have passed this on to the team. We will likely have something for this in the future. Thanks!
